apps/api/src/utils/error.ts

```
@@ -0,0 +1,192 @@
+    code: ErrorCode.SERVER_ERROR,
+    msg: '服务器内部错误',
+    data: null,
+  })
+}
+
+// 异步错误处理包装器
+export function asyncErrorHandler(fn: Function) {
+  return async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      await fn(req, res, next)
+    } catch (error) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)